modules/files/files.sh

```diff
@@ -1,38 +1,54 @@
 #!/usr/bin/env bash
 
 # Tell build process to exit if there are any errors.
-set -euo pipefail
+set -oue pipefail
```

Comment:
Why was this changed? I believe `o` has to be at the end for `pipefail` to work.

Reply:
Yeah, no need to change this

https://github.com/blue-build/modules/pull/99